apps/c_gpio/Makefile

```diff
@@ -0,0 +1,6 @@
+TOCK_APPS_DIR=..
```

Comment:
In light of not having any non-C apps, I propose we drop the `c_` prefix for apps from now on.